kohese-portal/tslint.json

```diff
@@ -1,143 +0,0 @@
-{
```

Comment:
Why was this file removed?

Reply:
tslint is fully deprecated in angular 13. angular.json no longer recognizes "lint" as a valid configuration option. I haven't checked for implementation, but it favors esLint now.